tests/e2e/codeflare_test.go

```diff
@@ -158,6 +179,30 @@ func (tc *CodeFlareTestCtx) validateCodeFlareReady() error {
 		return fmt.Errorf("error waiting Ready state for CodeFlare %v: %w", tc.testCodeFlareInstance.Name, err)
 	}
 
+	err = wait.PollUntilContextTimeout(tc.testCtx.ctx, generalRetryInterval, componentReadyTimeout, true, func(ctx context.Context) (bool, error) {
+		list := dscv1.DataScienceClusterList{}
+		err := tc.testCtx.customClient.List(ctx, &list)
+		if err != nil {
+			return false, err
+		}
+
+		if len(list.Items) != 1 {
+			return false, fmt.Errorf("expected 1 DataScience Cluster CR but found %v", len(list.Items))
+		}
+
+		for _, c := range list.Items[0].Status.Conditions {
+			if c.Type == componentApi.CodeFlareComponentName+"Ready" {
+				return c.Status == corev1.ConditionTrue, nil
+			}
+		}
+
+		return false, nil
+	})
+
+	if err != nil {
+		return fmt.Errorf("error waiting Ready state for CodeFlare componetn in DSC: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("error waiting on Ready state for CodeFlare component in DSC: %w", err)
```

and equivalent suggestion for the other tests that have this check

Reply:
done